### PR TITLE
Fix issue #10576 Viewer.destroy not checking if ScreenSpaceEventHandl…

### LINKED ISSUE #10576: Viewer.destroy not checking if ScreenSpaceEventHandler is defined
Sandcastle example:
https://sandcastle.cesium.com/#c=hVA9a8MwEP0rhztYhiDTuY4ppIUOgQ6BTl4U6ZqKyJIrnRzc0v9eOUogoYFqu/dx7+mks4Fg1HhAD0uweIAVBh17/nbEWFfI47xyloS26LuieuhsZ7OHB+kR7WYQEp9HtPQirDIJVxjIu4llsXIy9onlnxH9tEGDkpxn5d1JlsPKiguljmvWOhCmNFZKo+W+XACrYNnCd2cBZCrtDHLjduyfGjo85QRUrJq7wOmzVwV/cstiUTSBJoPtrJvfo+4H5wmiN4zzmrAfjCAM9TbKPRKXIeSlAE19aW2UHkGr5Y3zgTQihMS8R2M2+gu7om3qpP9jNU4obXevI3ojpln2cd+uM8g5b+o03naSc2YrUtiZ2UYiZzN5dfMkOV0IMtDUWXt2Xuz/BQ

If the viewers ScreenSpaceEventHandler has been destroyed prior to the viewer being destroyed, unminifed cesiumjs is throwing a developer error:
```The object is destroyed...```
The viewers destroy method is not checking if the handler has been destroyed prior to using it, like it is for other fields:
https://github.com/CesiumGS/cesium/blob/51f3c656e638dd2ead7a2e297f8a391bcb75b915/Source/Widgets/Viewer/Viewer.js#L1681

Happy to create a PR or remove the issue if it is desired behaviour. Although it does look like the viewer is only removing two input actions and not actually destroying the handler:
```
  this.screenSpaceEventHandler.removeInputAction(
    ScreenSpaceEventType.LEFT_CLICK
  );
  this.screenSpaceEventHandler.removeInputAction(
    ScreenSpaceEventType.LEFT_DOUBLE_CLICK
  );
```

### PR DESCRIPTION
**Pull request by nayangoyal**
Fixes #10576 

- Check if Screen Space Event Handler is defined or exists if yes then destroy it.

### CODE REVIEW

**Comment by ggetz** [comment]
Hi @nayangoyal, thanks for the contribution! 

Before we can review this PR, we'll need a [signed CLA](https://github.com/CesiumGS/cesium/blob/main/CONTRIBUTING.md#contributor-license-agreement-cla). Please submit one, and comment back here so we know to check.

**Comment by nayangoyal** [comment]
I just submitted it.

File: packages/widgets/Source/Viewer/Viewer.js [hunk 1]
```diff
@@ -1719,12 +1719,18 @@ Viewer.prototype.isDestroyed = function () {
...
   let i;
 
-  this.screenSpaceEventHandler.removeInputAction(
-    ScreenSpaceEventType.LEFT_CLICK
-  );
-  this.screenSpaceEventHandler.removeInputAction(
-    ScreenSpaceEventType.LEFT_DOUBLE_CLICK
-  );
+  // Check if Event handler is exist
+  if(define(this.screenSpaceEventHandler)){
```
Comment (ggetz) [inline packages/widgets/Source/Viewer/Viewer.js:1723]:
```suggestion
  if(defined(this.screenSpaceEventHandler)){
```

Comment (ggetz) [reply packages/widgets/Source/Viewer/Viewer.js:1723]: CI is also failing due to formatting issues. Can you run `npm run prettier` and submit the result?

**Review by ggetz — COMMENTED** [review]
Thanks for the fix @nayangoyal!

Would you please also:
1. Add your name to `CONTRIBUTORS.md`
2. Update `CHNAGES.md` with a summary of the fix

**Comment by ggetz** [comment]
Hi @nayangoyal! Any updates here?

**Comment by ggetz** [comment]
I'm closing this issue due to inactivity. If you believe this is still an issue, please feel free to re-open. Thanks!

**Pull request closed without merging**